pkg/cmd/cli/cmd/startbuild.go

```diff
@@ -167,7 +182,7 @@ func RunStartBuild(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra
 
 	var newBuild *buildapi.Build
 	switch {
-	case !isBuild && (len(fromFile) > 0 || len(fromDir) > 0 || len(fromRepo) > 0):
+	case resource != "builds" && (len(fromFile) > 0 || len(fromDir) > 0 || len(fromRepo) > 0):
```

Comment:
Should error out, will double check

On Oct 27, 2015, at 8:16 AM, Jordan Liggitt <notifications@github.com>
wrote:

In pkg/cmd/cli/cmd/startbuild.go
<https://github.com/openshift/origin/pull/4947#discussion_r43113528>:

> @@ -167,7 +182,7 @@ func RunStartBuild(f *clientcmd.Factory, in io.Reader, out io.Writer, cmd *cobra
>
>  	var newBuild *buildapi.Build
>  	switch {
> -	case !isBuild && (len(fromFile) > 0 || len(fromDir) > 0 || len(fromRepo) > 0):
> +	case resource != "builds" && (len(fromFile) > 0 || len(fromDir) > 0 || len(fromRepo) > 0):

@smarterclayton <https://github.com/smarterclayton> do fromFile, etc, get
ignored when resource == "builds"?

—
Reply to this email directly or view it on GitHub
<https://github.com/openshift/origin/pull/4947/files#r43113528>.